tests/testthat/test_prepareAnnotations.R

```diff
@@ -128,30 +128,25 @@ test_that("txid must be in EqClassById", {
 })
 
 
-# This function will pick a few genes, get their transcripts, and verify the 
+# This function will get the transcripts, and verify the 
```

Comment:
Since it is quite fast (58 sec for the hg38 genome), I think there's no need to test a portion of genes.

Reply:
It works for me, probably version difference of dplyr package I guess. But mine works with unlist added as well, so I will add it.

Reply:
I edited it here. 
[](https://github.com/GoekeLab/bambu/pull/276#discussion_r912629816)

Reply:
I'll send you the details to follow up on this

Reply:
this is resolved with the most recent dplyr version